delius-stage/sub-projects/new-tech.tfvars

```diff
@@ -27,6 +27,11 @@ pdfgenerator_conf = {
   ecs_scaling_max_capacity = 10
 }
 
+# Override default Offender Search Service Config
```

Comment:
stage is like preprod

Reply:
I thought stage was t2?

Reply:
Not in Delius land. Stage is a better pre-prod - it has more up to date data. I suspect at some point we will move all our apps to point at stage rather than preprod